helm/examples/azure/log-analytics/enable-loganalytics.sh

```diff
@@ -40,6 +40,23 @@ else
    echo "Tenant ID: ${tid}"
 fi
 
+echo "Confirming workspace resource group exists..."
+wsrgexist=$(az group exists -n $LOGANALYTICS_RESOURCE_GROUP)
+if [[ $? -ne 0 ]] || [[ $wsrgexist == false* ]]
+then
+   echo "creating $LOGANALYTICS_RESOURCE_GROUP resource group..."
+   if [[ -z "$AKS_RESOURCE_LOCATION" ]]
+   then
+     echo "Missing AKS_RESOURCE_LOCATION environment variable - this should be set to the desired Azure resource location"
+     exit 1
+   fi
+
+   azrgcreate=$(az group create -n $LOGANALYTICS_RESOURCE_GROUP -l $AKS_RESOURCE_LOCATION --tags $TAGS | awk '/id/ {print $2;}' | tr -d '"' | tr -d ',')
```

Comment:
if the az group create fails for any reason, this might give us funky output. We might want to run az group create first, save the output, check the return code and then continue with the formatting to get the resource id you output on line 55.

Reply:
Agreed, I'll add a simple check akin to line 62